### DISCUSSION TITLE
The hamburger and Actions are broken

### DISCUSSION THREAD

**Discussion by WoAnerges** [music-assistant/support#2614] [Q&A]
Bug 1: The hamburger menu button is broken in the folder browser. When you click the button near a folder, nothing happens. It does not open any menu. 

Bug 2: the multi selection tool is broken. When you select multiple folders and click the "Actions" button, nothing happens. It doesn't open the menu.

**Comment by OzGav** [comment]
Not sure why you weren’t able to find this https://github.com/music-assistant/hass-music-assistant/issues

You need to resubmit the issues there
